aten/src/ATen/native/SpectralOpsUtils.h

```diff
@@ -62,4 +63,18 @@ inline int64_t infer_ft_complex_to_real_onesided_size(int64_t complex_size,
   }
 }
 
+using fft_fill_with_conjugate_symmetry_fn =
+    void (*)(ScalarType dtype, IntArrayRef mirror_dims, IntArrayRef half_sizes,
+             IntArrayRef in_strides, const void * in_data,
```

Comment:
Style nit: void*, the pointer annotation defines the type (which is why we prefer not to declare multiple objects on the same line)